study-builder/config/testboston-vars.conf.ctmpl

```diff
@@ -0,0 +1,35 @@
+{{$environment := env "ENVIRONMENT"}}
+{{$version := env "VERSION"}}
+{{$study := env "STUDY_KEY"}}
+{{with $response := vault (printf "secret/pepper/%s/%s/%s/conf" $environment $version $study)}}
+{{with $conf := $response.Data}}
+
```

Comment:
I added a different `vars.conf.ctmpl` file because I'm assuming it's going to look a bit different from CMI studies.